src/Components/horizontal_card/Horizontal_Card.js

```diff
@@ -0,0 +1,37 @@
+import "./horizontal_card.css";
+const Horizontal_card=({productTitle, productSubtitle, productCost, productQty })=>{
+    return(
+        <div className="card card-horizontal">
+        <div className="card-img-container card-horizontal">
+          <img
+            className="card-img card-horizontal"
+            src="/Utils/images/image 1.png"
```

Comment:
We can import images at the top-level and use it in the src instead of providing path directly. Later, if you need to change the image or source, it would be easy for you instead of searching throughout the component.

Rest looks good to me, keep up!